src/WhereConditions/WhereConditionsServiceProvider.php

```diff
@@ -127,7 +130,9 @@ public static function createHasConditionsInputType(string $name, string $descri
             ->value;
         $operatorDefault = $operator->defaultHasOperator();
 
-        return Parser::inputObjectTypeDefinition(/** @lang GraphQL */ <<<GRAPHQL
+        return Parser::inputObjectTypeDefinition(
```

Comment:
I prefer the previous formatting

Reply:
Reverted formatting to the previous one.